src/SmartComponents/Advisor/Advisor.js

```diff
@@ -66,7 +65,7 @@ const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetch
                 }
                 {recStatsStatus !== 'fulfilled' ? <Loading /> :
                     <StackChartTemplate
-                        ariaDesc="Advisor recommendations"
+                        ariaDesc="This chart represents the incidents detecting your system."
```

Comment:
incidents a totally different thing, verbiage like... 'this chart shows recommendations effecting your system(s)'.